### PR TITLE
Update README and package.json

### PR DESCRIPTION
**Pull request by bg451**
### Plugin Checklist

- [ ] Unit tests.
- [ ] TypeScript [definitions][1].
- [ ] TypeScript [tests][2].
- [ ] API [documentation][3].
- [ ] CircleCI [jobs/workflows][4].
- [ ] Plugin is [exported][5].

[1]: https://github.com/DataDog/dd-trace-js/blob/master/index.d.ts
[2]: https://github.com/DataDog/dd-trace-js/blob/master/docs/test.ts
[3]: https://github.com/DataDog/dd-trace-js/blob/master/docs/API.md
[4]: https://github.com/DataDog/dd-trace-js/blob/master/.circleci/config.yml
[5]: https://github.com/DataDog/dd-trace-js/blob/master/packages/dd-trace/src/plugins/index.js

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,26 +1,26 @@
...
 [![BrowserStack Status](https://automate.browserstack.com/badge.svg?badge_key=TU95QWlIQXhOcGw2YkdvVGpSYkNLK2QveGlwbmRYc3FSVFRtMUcza3hhQT0tLWErRVVDMFMvWnVIU3p5OE9ZSFJWeXc9PQ==--f63f623010664e0a1776325aefd8d119362f31d4)](https://automate.browserstack.com/public-build/TU95QWlIQXhOcGw2YkdvVGpSYkNLK2QveGlwbmRYc3FSVFRtMUcza3hhQT0tLWErRVVDMFMvWnVIU3p5OE9ZSFJWeXc9PQ==--f63f623010664e0a1776325aefd8d119362f31d4)
 
+Datadog has generously announced the [donation](https://www.datadoghq.com/blog/opentelemetry-instrumentation) of their tracer libraries to the [OpenTelemety](https://opentelemetry.io/), project. Auto-instrumentation is a core feature of these libraries, making it possible to create and collect telemetry data without needing to change your code. LightStep wants you to be able to use these libraries now! `ls-trace-dontnet` is LightStep's fork of Datadog’s tracing client for .NET. You can install and use it to take advantage of auto-instrumentation without waiting for OpenTelemetry. Each LightStep agent is [“pinned” to a Datadog release](#versioning) and is fully supported by LightStep’s Customer Success team.
```
Comment (codeboten) [inline README.md]:
```suggestion
Datadog has generously announced the [donation](https://www.datadoghq.com/blog/opentelemetry-instrumentation) of their tracer libraries to the [OpenTelemety](https://opentelemetry.io/), project. Auto-instrumentation is a core feature of these libraries, making it possible to create and collect telemetry data without needing to change your code. LightStep wants you to be able to use these libraries now! `ls-trace-js` is LightStep's fork of Datadog’s tracing client for Javascript. You can install and use it to take advantage of auto-instrumentation without waiting for OpenTelemetry. Each LightStep agent is [“pinned” to a Datadog release](#versioning) and is fully supported by LightStep’s Customer Success team.
```

Comment (bg451) [reply README.md]: oop good catch, thanks!

File: package.json [hunk 2]
```diff
@@ -1,7 +1,7 @@
 {
-  "name": "dd-trace",
-  "version": "0.16.0",
-  "description": "Datadog APM tracing client for JavaScript",
+  "name": "ls-trace",
+  "version": "0.1.0",
+  "description": "Datadog/LS APM tracing client for JavaScript",
```
Comment (codeboten) [inline package.json]:
```suggestion
  "description": "LightStep fork of Datadog's APM tracing client for JavaScript",
```

File: package.json [hunk 3]
```diff
@@ -41,9 +42,9 @@
   "author": "Datadog Inc. <info@datadoghq.com>",
```
Comment (codeboten) [inline package.json]:
```suggestion
  "author": "LightStep <support@lightstep.com>",
```

**Review by codeboten — CHANGES_REQUESTED** [review]
Just a couple of suggestions to remain consistent with other packages

**Review by codeboten — APPROVED** [review]

**Pull request merged**